libraries/AP_HAL_ChibiOS/hwdef/include/minimize_common.inc

```diff
@@ -48,6 +48,7 @@ define AP_WINCH_ENABLED 0
 define AP_CAMERA_BACKEND_DEFAULT_ENABLED 0
 define AP_CAMERA_RELAY_ENABLED AP_CAMERA_ENABLED
 define AP_CAMERA_SERVO_ENABLED AP_CAMERA_ENABLED
+define AP_CAMERA_RUNCAM_ENABLED AP_CAMERA_ENABLED
```

Comment:
This doesn't look right.  We shouldn't be adding things to minimized boards like this - we only have these other backends in here so we don't break people's builds!

Reply:
RunCam is already enabled in the minimized builds, this just makes sure it continues to be so